autobotAI_integrations/utils/__init__.py

```
@@ -431,9 +439,9 @@ def extract(resources: List[Any], selector: str, logger) -> List[Dict[str, Any]]
                 else:
                     results.append({str(match.path): match.value, **common_fields})
 
-        return results if results else resources
+        return results
 
-    except (JSONPathError, Exception):
+    except (JSONPathError, Exception) as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)